### PR TITLE
1.7.10

### PR DESCRIPTION
**Pull request by ENDERZOMBI102**
map many things
i forgot what i've mapped, but its a lot

### CODE REVIEW

File: mappings/net/minecraft/CoordComparator.mapping [hunk 1]
```diff
@@ -0,0 +1,14 @@
+CLASS net/minecraft/class_2349 net/minecraft/CoordComparator
+	COMMENT TODO: Use better name
+	FIELD field_10572 x F
+	FIELD field_10573 y F
+	FIELD field_10574 z F
+	FIELD field_10575 buffer [I
+	METHOD <init> ([IFFF)V
+		ARG 1 buffer
```
Comment (hYdos) [inline mappings/net/minecraft/CoordComparator.mapping:8]: Why is this code taking a buffer in to compare Coordinates?

File: mappings/net/minecraft/IntCache.mapping [hunk 2]
```diff
@@ -0,0 +1,11 @@
+CLASS net/minecraft/class_604 net/minecraft/IntCache
```
Comment (hYdos) [inline mappings/net/minecraft/IntCache.mapping:1]: You have forgotten to put any of these classes in packages.

**Review by hYdos — CHANGES_REQUESTED** [review]

File: mappings/net/minecraft/IntCache.mapping (hunk 2 above)
Comment (ENDERZOMBI102) [reply mappings/net/minecraft/IntCache.mapping:1]: where would you put this one?

Comment (hYdos) [reply mappings/net/minecraft/IntCache.mapping:1]: Unsure based on the class name alone. What does it do internally and where is it used?

File: mappings/net/minecraft/CoordComparator.mapping (hunk 1 above)
Comment (ENDERZOMBI102) [reply mappings/net/minecraft/CoordComparator.mapping:8]: i have no idea, but it takes a int array of size 3 together with xyz float components

File: mappings/net/minecraft/IntCache.mapping (hunk 2 above)
Comment (ENDERZOMBI102) [reply mappings/net/minecraft/IntCache.mapping:1]: it seems to be some sort of allocator/cache based on ints/int arrays

File: mappings/net/minecraft/CoordComparator.mapping (hunk 1 above)
Comment (ENDERZOMBI102) [reply mappings/net/minecraft/CoordComparator.mapping]: i'll mark this as resolved, but feel free to ask again if you still have dubts about the name

**Review by arthurbambou — APPROVED** [review]

**Review by arthurbambou — APPROVED** [review]

**Pull request closed without merging**